### ISSUE TITLE
Please add/return similar signs

### ISSUE THREAD

**Issue by art-shen** [symbl-cc/symbl-data#182]
In the old version, I used to find many symbols through the similarity. Seems now it's not presented anywhere.
Like finding all variants of dashes through the `-` symbol.

Btw, a separated typography collection would be really nice to have too. Currently, it's hard to find symbols for typography.

**Comment by asanov** [comment]
There are related symbols at the bottom of the page:
![image](https://github.com/symbl-cc/symbl-data/assets/3072322/29f3429a-cb64-44f5-a8b4-cdeb60050521)
And good idea about typography symbols.

**Comment by amal** [comment]
@asanov I don't see this block at all.

E.g. for `-`: https://symbl.cc/en/002D/

![image](https://github.com/symbl-cc/symbl-data/assets/423417/0cc8afb4-356f-4933-9f0d-e79e088f5884)

**Comment by asanov** [comment]
@amal yes, thank you. This means that the symbol has no related symbols. We will check why. The functionality of related symbols has not yet been worked out. We have it in the future development plan.
